test/js/ez-selection-activate-element.js

```diff
@@ -0,0 +1,117 @@
+describe('ez-selection-activate-element', function() {
+    let element;
+    let elementSubsetElementChecked;
+
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+        elementSubsetElementChecked =  fixture('TestFixtureSubsetElementChecked');
+    });
+
+    function checkAndclickOneElement(elem, selector, isChecked) {
+        const checkbox = elem.querySelector(selector);
+        // we need to prevent the click event at the document level
+        // to prevent Edge/Safari from checking the input when clicked
+        // so preventDefault() will be called in the test but we verify
+        // that's it not called, that's why `preventDefaultCalled`
+        // local variable was introduced.
+        let preventDefaultCalled = false;
+        const prevent = function(e) {
+            preventDefaultCalled = e.defaultPrevented;
+            e.preventDefault();
+        };
+
+        document.addEventListener('click', prevent);
+        checkbox.checked = isChecked;
+        checkbox.dispatchEvent(new CustomEvent('click', {
+            bubbles: true,
+        }));
+
+        assert.isFalse(preventDefaultCalled);
+    }
+
+    function checkAndclickEveryElements(elem, selector, isChecked) {
+        const checkboxes = elem.querySelectorAll(selector);
+        // same comment than in checkAndclickOneElement().
+        let preventDefaultCalled = false;
+        const prevent = function(e) {
+            preventDefaultCalled = e.defaultPrevented;
+            e.preventDefault();
+        };
+
+        document.addEventListener('click', prevent);
+        checkboxes.forEach((checkbox) => {
+            checkbox.checked = isChecked;
+            checkbox.dispatchEvent(new CustomEvent('click', {
+                bubbles: true,
+            }));
+        });
+
+        assert.isFalse(preventDefaultCalled);
+    }
+
+    it('should be defined', function () {
+        assert.equal(
+            window.customElements.get('ez-selection-activate-element'),
+            element.constructor
+        );
+    });
+
+    describe('properties', function () {
+        describe('`selectionSelector`', function () {
+            it('should default to `input[type="checkbox"]``', function () {
+                assert.equal(element.selectionSelector, 'input[type="checkbox"]');
+            });
+        });
+    });
+
+    describe('activate elements', function () {
+        describe('single selection', function () {
+            it('should enable activable buttons', function () {
+                const activableElement = element.querySelector('.ez-js-activable-element');
+
+                checkAndclickOneElement(element, element.selectionSelector, true);
+                assert.isFalse(activableElement.disabled);
+            });
+
+            it('should enable activable buttons based on custom selection', function () {
```

Comment:
you should have `describe('custom selection')` to group this test and the following one